libraries/math/src/approximations.rs

```diff
@@ -41,6 +41,31 @@ pub fn sqrt<T: PrimInt + CheckedShl + CheckedShr>(radicand: T) -> Option<T> {
     Some(result)
 }
 
+/// Calculate the normal cdf of the given number
+///
+/// Code lovingly adapted from the excellent work at:
+///
+/// <https://www.hrpub.org/download/20140305/MS7-13401470.pdf>
+///
+/// The algorithm is based on the implementation in the paper above.
+#[inline(never)]
+pub fn f32_normal_cdf(argument: f32) -> Option<f32> {
+    const PI: f32 = std::f32::consts::PI;
+
+    let mod_argument = if argument < 0.0 {
+        -1.0 * argument
+    } else {
+        argument
+    };
+    let tabulation_numerator: f32 =
+        (1.0 / (1.0 * (2.0 * PI).sqrt())) * (-1.0 * (mod_argument * mod_argument) / 2.0).exp();
```

Comment:
It looks like the paper is inconsistent about this, and in the "calculate implementation" also suggests:
```suggestion
        (1.0 / (2.0 * (2.0 * PI).sqrt())) * (-1.0 * (mod_argument * mod_argument) / 2.0).exp();
```
Any idea which is right?

Reply:
Any response here?